src/view/pages/gettingStarted.tsx

```diff
@@ -69,12 +106,20 @@ export class GettingStartedPage extends React.Component {
                             <pre> if button_b.is_pressed():</pre>
                             <pre> display.show(Image.SAD)</pre>
                         </span>
-                        <h3> 4. Read then display the temperature.</h3>
+                        <h3> 4. Read then display the temperature</h3>
                         <span className="codeBox">
                             <pre>while True:</pre>
                             <pre> temp = temperature()</pre>
                             <pre> display.show(temp)</pre>
                         </span>
+                        <h3>
+                            {" "}

```

Comment:
Was the space added in the h3 on purpose?

Reply:
Was an accident. Removed!